exchange/exchange-ps/exchange/Get-AuthenticationPolicy.md

```diff
@@ -81,10 +81,13 @@ The AllowLegacyExchangeTokens switch specifies whether legacy Exchange tokens ar
 
 Legacy Exchange tokens include Exchange user identity and callback tokens.
 
+This switch also specifies the most recent date and time that an add-in was allowed or blocked from acquiring a token.
+
 **Important**:
 
-- Currently, the AllowLegacyExchangeTokens switch only specifies whether legacy Exchange tokens are allowed in your organization. For now, disregard the empty Allowed and Blocked arrays returned by the switch.
-- Legacy Exchange tokens will eventually be blocked by default in all cloud-based organizations. For more information, see [Nested app authentication and Outlook legacy tokens deprecation FAQ](https://learn.microsoft.com/office/dev/add-ins/outlook/faq-nested-app-auth-outlook-legacy-tokens#what-is-the-timeline-for-shutting-down-legacy-exchange-online-tokens).
+- As of February 10, 2025, the AllowLegacyExchangeTokens switch specifies any add-in that requested an Exchange token.
```

Comment:
> any add-in

I wonder if we should clarify that we're talking about 3rd party add-ins... E.g. Viva may still get those tokens (and people may see that in fiddler) but it's not gonna be listed. That may cause them to think telemetry is inaccurate, but the fact is Viva is simply excluded from the whole thing for now as they work on migration. Please check with Paul on whether we want to say anything here about this or not.